Lib/test/test_genericalias.py

```diff
@@ -47,46 +47,46 @@
 
 class BaseTest(unittest.TestCase):
     """Test basics."""
+    types = [type, tuple, list, dict, set, frozenset, enumerate,
```

Comment:
I'd be more specific (in the variable name) what kinds of types are stored here ('generic_types', 'subscriptable_types' or something similar?)